### ISSUE TITLE
null default behvaviour not consistent/reliable

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/conforma-web-app#1483]
Will add more details later, just noting this so it gets taken care of.

**Comment by jagoje** [comment]
@CarlosNZ - Any update for this one. I think this has been dealt with?

**Comment by CarlosNZ** [comment]
> @CarlosNZ - Any update for this one. I think this has been dealt with?

No, it's an edge case, need to look into it at some point. Shouldn't affect any of the current workflows.

Feel free to move it out of the Fiji board, but I'll keep the issue open.
